Recruitment/JobSpecs/why_would_you_join_us.md

```diff
@@ -10,6 +10,6 @@ These are the things that drive us. Do they drive you?
 
 We are looking for people who want to help us change the way companies think and approach problems; people who want to work with teams and individuals who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or customer experience.
 
-We are looking for people who want to be treated like adults; people who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it; people who are comfortable finding their own place.
+We are looking for people who want to be treated like adults; people who neither need, nor want, someone else to tell them what needs to be done, but instead who can look at what’s happening, what’s needed, and figure out how they fit into it; people who are comfortable finding their own place; people who can be egoless about the work they produce.

```

Comment:
I didn't know that. Shame, I've had more than one candidate say that this is what they like the most from the previous JD. I know I would think the same and I know exactly what I mean by it. But if the group decided to drop it, then let's leave it at that.